l10n/en/products/vpn/platforms/shared.ftl

```diff
@@ -0,0 +1,35 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/products/vpn/desktop/
```

Comment:
Should this be https://www-dev.allizom.org/products/vpn/?

Reply:
Good call – it applies to multiple pages right now. Should I list each one?

Reply:
If it is section based, then you have the high level URL at the top. At the lower part, broken down in sections, then you can add the URL ahead of the section, no line break in that section. This means it applies to all the strings in that section. I think that's how it works.

Reply:
Cool. Updated per your comment. Thanks!